src/components/icon-button/icon-button.jsx

```diff
@@ -12,6 +12,7 @@ export default function IconButton(props) {
       fill="skip"
       icon={props.icon}
       onClick={props.onClick}
+      onEnter={props.onEnter}
```

Comment:
Honestly, I would prefer to reuse `onClick`. There shouldn't ever be a reason to diverge both interactions. And if that is the case, we want the developer to deal with the logic in their own domain and base it off the synthetic event.